ehrapy/tools/_sa.py

```diff
@@ -126,7 +127,10 @@ def kmf(
     weights: Iterable | None = None,
     censoring: Literal["right", "left"] = None,
 ) -> KaplanMeierFitter:
-    """Fit the Kaplan-Meier estimate for the survival function.
+    """
+    DEPRECATION WARNING: This function is deprecated and will be removed in the next release. Use `kaplan_meier` instead.
+
+    Fit the Kaplan-Meier estimate for the survival function.
```

Comment:
Somewhat self evident. Do we need that comment? I'm not sure...

Reply:
left it for now, as its across all survival analysis functions..